packages/compat/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "1.0.3",
   "description": "Compatibility utilities for ESLint",
   "type": "module",
+  "main": "dist/esm/index.js",
```

Comment:
I noticed that for `@eslint/eslintrc` alone we are exposing `cjs` as main and at rest of the places its esm. Is there some reason behind it? 

https://github.com/eslint/eslintrc/blob/fc9837d3c4eb6c8c97bd5594fb72ea95b6e32ab8/package.json#L6

Reply:
Yes. `@eslint/eslintrc` was made primarily for a CJS environment. Going forward, we are targeting primarily ESM. Everything in this repo is ESM-first.